dcmdump/src/main.rs

```diff
@@ -52,7 +84,17 @@ where
     }
 }
 
+#[cfg(windows)]
+fn os_compatibility() {
+    control::set_virtual_terminal(true);
```

Comment:
Look out, this method returns a `Result` which should be handled. Try propagating it to this function and using the existing error report function.

Reply:
It looks like the [original method returns `Result<(), ()>`](https://github.com/mackwic/colored/blob/master/src/control.rs#L28), so I wasn't sure how to use the existing `report` function since that expects an error. I added an exit code instead, but let me know if there's a better approach!